doc/src/learn/sui-compared.md

```diff
@@ -123,6 +123,20 @@ Sui mitigates this with checkpoints. A checkpoint is established every time an i
 
 Sui uses the state commitment that arrives upon epoch change. Sui requires a single answer from the multiple validators and leverages an accessory protocol to derive the hash representing the state of the blockchain. This protocol consumes little bandwidth and does not impede the ingestion of transactions. Validators produce checkpoints at every epoch change. Sui requires the validators to also produce checkpoints even more frequently. So users may use these checkpoints to audit the blockchain with some effort.
 
+## Summary
+
+|                 **Concept**                 |                  **Sui**                  |  **Traditional blockchain** |
+|:-------------------------------------------:|:-----------------------------------------:|:---------------------------:|
+| _Unit of storage_                           | object                                    | account                     |
+| _Transaction execution_                     | parallel (for independent transactions)   | sequential                  |
+| _Transactions flow_                         | proactively talk with validators          | fire-and-forget broadcast   |
```

Comment:
not sure what is meant by proactively talk with validators. Sui is inherently based on Byzantine Fault-Tolerant Broadcast as well?

Reply:
I took it from the docs. "How Sui works" page:
![image](https://user-images.githubusercontent.com/32844693/216573095-76886386-ea69-49c8-a438-7e1d447bc9d0.png)

and also "Simple transaction" section:
![image](https://user-images.githubusercontent.com/32844693/216573530-c1554a46-4c98-4927-8623-6689fc7eb663.png)